desktop-exporter/app/routes/main-view.tsx

```diff
@@ -29,57 +36,64 @@ function Row({ index, style, data }: RowProps) {
 }
 
 type SidebarProps = {
-  isClosed: boolean;
+  isFullWidth: boolean;
   toggle: () => void;
 };
 
 function Sidebar(props: SidebarProps) {
-  if (props.isClosed) {
-    return (
-      <div className="sidebar closed">
-        <button
-          className="menuBtn"
-          onClick={props.toggle}
-        >
-          Expand
-        </button>
-      </div>
+  const sidebarColour = useColorModeValue("pink.100", "pink.900");
+  const { traceSummaries } = useLoaderData() as TraceSummaries;
+
+  let sidebarWidth = "80px";
+  let buttonIcon = <ArrowRightIcon />;
+  let traceList = <></>;
+
+  if (props.isFullWidth) {
+    sidebarWidth = "250px";
+    buttonIcon = <ArrowLeftIcon />;
+    traceList = (
+      <FixedSizeList
+        className="list"
+        height={500}
+        itemData={traceSummaries}
+        itemCount={traceSummaries.length}
+        itemSize={30}
+        width="100%"
+      >
+        {Row}
+      </FixedSizeList>
     );
   }
 
-  const { traceSummaries } = useLoaderData() as TraceSummaries;
   return (
-    <div className="sidebar">
-      <button
-        className="menuBtn"
-        onClick={props.toggle}
-      >
-        Collapse
-      </button>
-      <nav>
-        <FixedSizeList
-          className="list"
-          height={500}
-          itemData={traceSummaries}
-          itemCount={traceSummaries.length}
-          itemSize={30}
-          width={"100%"}
-        >
-          {Row}
-        </FixedSizeList>
-      </nav>
-    </div>
+    <Flex
+      bgColor={sidebarColour}
+      direction="column"
+      transition="width 0.2s ease-in-out"
+      width={sidebarWidth}
+    >
+      <Flex justifyContent={"flex-end"}>
```

Comment:
nit: if you are going to use a string, you don't need the curly brackets

```suggestion
      <Flex justifyContent="flex-end">
```

Reply:
Autocomplete was trying to be helpful 😭